### PR TITLE
Fixes for odoo pylint

### PR DESCRIPTION
**Pull request by vwvw**
Fixes for a lot of pylint issues comming up in travis. 

Added blank lines at end of files.
Created ReadMe.rst
Fixed local imports
Changed tab format.

### CODE REVIEW

**Comment by ecino** [comment]
Good job, but there are still some errors :(

**Comment by vwvw** [comment]
Yeah, but for the other errors we need to discuss to see what's wrong ;-)

**Comment by ecino** [comment]
Tests are failing. Conflicts with latest changes.

**Comment by ecino** [comment]
Pylint is good now but something must be broken, the tests are failing :
```
ailed to initialize database `openerp_test`.
Traceback (most recent call last):
  File "/home/travis/odoo-10.0/odoo/service/server.py", line 911, in preload_registries
    registry = Registry.new(dbname, update_module=update_module)
  File "/home/travis/odoo-10.0/odoo/modules/registry.py", line 83, in new
    odoo.modules.load_modules(registry._db, force_demo, status, update_module)
  File "/home/travis/odoo-10.0/odoo/modules/loading.py", line 339, in load_modules
    loaded_modules, update_module)
  File "/home/travis/odoo-10.0/odoo/modules/loading.py", line 237, in load_marked_modules
    loaded, processed = load_module_graph(cr, graph, progressdict, report=report, skip_modules=loaded_modules, perform_checks=perform_checks)
  File "/home/travis/odoo-10.0/odoo/modules/loading.py", line 136, in load_module_graph
    registry.init_models(cr, model_names, {'module': package.name})
  File "/home/travis/odoo-10.0/odoo/modules/registry.py", line 318, in init_models
    model.init()
  File "/home/travis/build/CompassionCH/compassion-switzerland/sponsorship_switzerland/reports/sponsorships_evolution_months_report.py", line 72, in init
    self.env.cr.execute(query, params)
  File "/home/travis/odoo-10.0/odoo/sql_db.py", line 154, in wrapper
    return f(self, *args, **kwargs)
  File "/home/travis/odoo-10.0/odoo/sql_db.py", line 231, in execute
    res = self._obj.execute(query, params)
ProgrammingError: syntax error at or near "'sponsorships_evolution_months_report'"
LINE 2:             CREATE OR REPLACE VIEW 'sponsorships_evolution_m...
                                           ^
```

**Comment by vwvw** [comment]
Travis is green !

File: partner_communication_switzerland/views/recurring_contracts_view.xml [hunk 1]
```diff
@@ -9,4 +9,62 @@
             </field>
         </field>
     </record>
-</odoo>
\ No newline at end of file
+    <record id="invoice_line_inherit_muskathlon" model="ir.ui.view">
```
Comment (ecino) [inline partner_communication_switzerland/views/recurring_contracts_view.xml]: All those muskathlon records have been moved to the module `muskathlon`. You can remove them

File: pgsql_auto_backup/i18n/bg.po [hunk 2]
```diff
@@ -35,7 +35,7 @@ msgid "Invalid model name in the action definition."
 msgstr "Невалидно име на модел при задаване на действие"
 
 #. module: pgsql_auto_backup
-#: model:ir.model,name:auto_backup.model_db_backup
+#: model:ir.model,name:auto_backup.model_DbBackup
```
Comment (ecino) [inline pgsql_auto_backup/i18n/bg.po]: I think this is a bad search/replace... you can undo your changes in the .po files

**Review by ecino — CHANGES_REQUESTED** [review]
It's a historical day for Travis ;)

**Review by ecino — APPROVED** [review]

**Pull request merged**